content/02.body.md

```diff
@@ -13,6 +13,17 @@
 
 ### Network permutation
 
+Network permutation produces new networks that do not share their edges with the original network.
```

Comment:
This is not true. See [Cell 4 figure here](https://nbviewer.jupyter.org/github/dhimmel/integrate/blob/ebd71cd2157d26e52646b5b483f5c70293a84f71/data/permuted/evaluate-permutations.ipynb). Some edges will still be shared, but the same amount that other random networks share with each other.

Reply:
Agree, that was poorly phrased.